README.md

```diff
@@ -18,6 +18,8 @@ and support full JetStream management.
 
 ### Installation
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/natscli.svg)](https://repology.org/project/natscli/versions)
```

Comment:
since this really only apply to Arch, I think it's best to remove this line, else this is good thanks :)